src/components/LocationPage/ShareButtons.style.ts

```diff
@@ -1,31 +1,13 @@
 import styled, { css } from 'styled-components';
+import MuiButton from '@material-ui/core/Button';
 import palette from 'assets/theme/palette';
 import { StyledShareButtonStyles } from '../ShareBlock/ShareBlock.style';
 import { COLOR_MAP } from 'common/colors';
 import MuiCircularProgress from '@material-ui/core/CircularProgress';
 
-const WrapperStyles = css`
-  display: block;
+export const Wrapper = styled.div`
   position: relative;
-  height: 0;
-  z-index: 1;
-`;
-
-export const DesktopButtonsWrapper = styled.div`
```

Comment:
now that the share buttons render in the same place on both mobile+desktop, we dont need these wrappers